cpp/modmesh/math/pymod/wrap_Complex.cpp

```diff
@@ -52,31 +53,42 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapComplex
     {
         namespace py = pybind11; // NOLINT(misc-unused-alias-decls)
 
+        PYBIND11_NUMPY_DTYPE(wrapped_type, real_v, imag_v);
+
         (*this)
-            .def(
-                py::init(
-                    [](const value_type & real_v, const value_type & imag_v)
-                    { return std::make_shared<wrapped_type>(real_v, imag_v); }),
-                py::arg("real_v"),
-                py::arg("imag_v"))
             .def(
                 py::init(
                     []()
                     { return std::make_shared<wrapped_type>(); }))
+            .def(
+                py::init(
+                    [](const value_type & real, const value_type & imag)
+                    { return std::make_shared<wrapped_type>(wrapped_type{.real_v = real, .imag_v = imag}); }),
+                py::arg("real"),
+                py::arg("imag"))
             .def(
                 py::init(
                     [](const wrapped_type & other)
-                    { return std::make_shared<wrapped_type>(other); }),
+                    { return std::make_shared<wrapped_type>(wrapped_type{.real_v = other.real_v, .imag_v = other.imag_v}); }),
                 py::arg("other"))
             .def(py::self + py::self) // NOLINT(misc-redundant-expression)
             .def(py::self - py::self) // NOLINT(misc-redundant-expression)
             .def(py::self * py::self) // NOLINT(misc-redundant-expression)
+            .def(py::self / py::self) // NOLINT(misc-redundant-expression)
             .def(py::self / value_type()) // NOLINT(misc-redundant-expression)
             .def(py::self += py::self) // NOLINT(misc-redundant-expression)
             .def(py::self -= py::self) // NOLINT(misc-redundant-expression)
-            .def_property_readonly("real", &wrapped_type::real)
-            .def_property_readonly("imag", &wrapped_type::imag)
+            .def(py::self *= py::self) // NOLINT(misc-redundant-expression)
+            .def(py::self /= py::self) // NOLINT(misc-redundant-expression)
+            .def(py::self /= value_type()) // NOLINT(misc-redundant-expression)
+            .def("__lt__", &wrapped_type::operator<)
+            .def("__gt__", &wrapped_type::operator>)
+            .def_readonly("real", &wrapped_type::real_v)
+            .def_readonly("imag", &wrapped_type::imag_v)
             .def("norm", &wrapped_type::norm)
+            .def("dtype",
+                 []()
+                 { return py::dtype::of<wrapped_type>(); })
```

Comment:
Add `dtype` getter is convenient for user to assign numpy `dtype` while initial numpy arrary.